compiler/rustc_typeck/src/check/coercion.rs

```diff
@@ -87,6 +89,29 @@ impl<'a, 'tcx> Deref for Coerce<'a, 'tcx> {
 
 type CoerceResult<'tcx> = InferResult<'tcx, (Vec<Adjustment<'tcx>>, Ty<'tcx>)>;
 
+struct CollectRetsVisitor<'tcx> {
+    ret_vec: Vec<&'tcx hir::Expr<'tcx>>,
```

Comment:
We rarely name the fields or variables after the type they hold, usually we differentiate if it's a vector or slice by using the plural. What do you think of renaming as follows?
```suggestion
    ret_exprs: Vec<&'tcx hir::Expr<'tcx>>,
```

Reply:
ret_exprs works for me and the change was made